Sigh... Why do I keep forgetting to run the Maven tests before committing! Anyway, the error seemed to have been caused because I didn't put a blank line before the list. That that was the problem was pretty well hidden, though:
```
[INFO] org.openhab.binding.pihole/README.md:33 error MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "- Version 5: Go to `settings` ..."]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for openHAB Add-ons 5.2.0-SNAPSHOT:
[INFO] 
[INFO] openHAB Add-ons .................................... SUCCESS [  5.900 s]
[INFO] openHAB Add-ons :: BOM ............................. SUCCESS [  0.046 s]
[INFO] openHAB Add-ons :: BOM :: Runtime Index ............ SUCCESS [  3.007 s]
[INFO] openHAB Add-ons :: BOM :: Test Index ............... SUCCESS [  1.543 s]
[INFO] openHAB Add-ons :: BOM :: openHAB Core Index ....... SUCCESS [ 12.860 s]
[INFO] openHAB Add-ons :: Bundles ......................... FAILURE [ 41.078 s]
[INFO] openHAB Add-ons :: Bundles :: Pi-hole Binding ...... SKIPPED
[INFO] openHAB Add-ons :: BOM :: openHAB Add-ons .......... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  50.389 s (Wall Clock)
[INFO] Finished at: 2026-05-06T13:38:37Z
[INFO] ------------------------------------------------------------------------
[INFO] Static code analysis summary report is available in:
[INFO] file:/home/runner/work/openhab-addons/openhab-addons/target/summary_report.html
Error:  Failed to execute goal com.github.eirslett:frontend-maven-plugin:2.0.0:npm (markdownlint) on project org.openhab.addons.reactor.bundles: Failed to run task: 'npm exec --yes markdownlint-cli2 -- **/*.md !**/target/** !**/node_modules/** --config /home/runner/work/openhab-addons/openhab-addons/.github/markdownlint.yaml' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :org.openhab.addons.reactor.bundles
Error: Process completed with exit code 1.
```

--- a/bundles/org.openhab.binding.pihole/README.md
+++ b/bundles/org.openhab.binding.pihole/README.md
@@ -21,8 +21,8 @@ The Pi-hole Binding allows you to monitor Pi-hole statistics and control its fun
 
 ### `server` Thing Configuration
 
-| Name            | Type    | Description                                                                                                                                                                                  | Default | Required | Advanced |
-| --------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- | -------- | -------- |
+| Name            | Type    | Description                                           | Default | Required | Advanced |
+| --------------- | ------- | ----------------------------------------------------- | ------- | -------- | -------- |
 | hostname        | text    | URL (hostname or IP address) of the pihole web server | N/A     | yes      | no       |
 | token           | text    | Token to access the device (see below)                | N/A     | yes      | no       |
 | refreshInterval | integer | Interval the device is polled in sec                  | 600     | no       | yes      |

--- a/bundles/org.openhab.binding.pihole/README.md
+++ b/bundles/org.openhab.binding.pihole/README.md
@@ -30,6 +30,7 @@ The Pi-hole Binding allows you to monitor Pi-hole statistics and control its fun
 ### Generating a token
 
 The procedure for generating a token in version 5 is different from generating one in version 6:
+
 - Version 5: Go to `settings` > `API` > `Show API token`
 - Version 6: Go to `settings` > `Web Interface - API Settings` > `Expert` > `Configure app password` > Copy the `app password` > `Enable new app password`
 

